### PR TITLE
 [ES] Update `Beatmap discussion`

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

**Comment by UnknownOsu** [comment]
Tal vez puedas quitar el `outdated` de arriba

**Comment by TicClick** [comment]
@agatemosu hey, what's the status of this PR?

File: wiki/Beatmap_discussion/es.md [hunk 1]
```diff
@@ -23,17 +23,17 @@ La **discusión de beatmaps** (también conocida como *Modding v2*) es un sistem
 - Campo de envío
 - Discusiones
 
-¡Puedes seguir nuestras imágenes de [este mapa](https://osu.ppy.sh/beatmapsets/924551/discussion) para familiarizarte con las discusiones de beatmaps o intentar abrir una página de discusión tú mismo!
+¡Puedes seguir nuestras imágenes de [este beatmap](https://osu.ppy.sh/beatmapsets/924551/discussion) para familiarizarte con las discusiones de beatmaps o intentar abrir una página de discusión tú mismo!
 
 ## Inicio rápido
 
-Es importante tener en cuenta algunas cosas para usar las páginas de discusiones de beatmaps de manera efectiva:
+Es importante tener en cuenta algunas cosas para usar las páginas de discusión de beatmaps de manera efectiva:
```
[the remark this answers is not in the document]
Comment (UnknownOsu) [reply wiki/Beatmap_discussion/es.md:30]: @agatemosu

**Comment by agatemosu** [comment]
its ok

**Review by UnknownOsu — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**